src/aggregators/aggregated_api.jl

```diff
@@ -1,5 +1,5 @@
 function reset_aggregated_jumps!(integrator,uprev = nothing)
-     reset_aggregated_jumps!(integrator,uprev,integrator.opts.callback)
+     reset_aggregated_jumps!(integrator,uprev,integrator.cb)

```

Comment:
This isn't correct. It's in opts.callback, and it's not a no-op if there's jumps